spec/game_spec.rb

```diff
@@ -1,20 +1,105 @@
 # frozen_string_literal: true
 
 require 'game'
+require 'board'
+require 'player'
+require 'players'
+require 'console'
+require 'stringio'
+
+describe Game do
+  before do
+    @console = Console.new
+    @players = Players.new([Player.new('Player 1', 'X'), Player.new('Player 2', 'O')])
+    $stdout = StringIO.new
+  end
 
-describe 'Game' do
   context 'when starting a new game' do
-    let(:game) { Game.new }
+    before do
+      @board = Board.new
+      @game = Game.new(@board, @players, @console)
+    end
+
+    it "initializes as player 1's turn by default" do
+      expect(@game.current_player.name).to eq('Player 1')
+    end
```

Comment:
From looking at your `Game` class, I noticed that `current_player` is a read attribute and it looks like it's because it's being used in this class. Rather, could you just check that the game initializes as player 1's turn by asserting on the output? Lmk if you wanna pair on this